### DISCUSSION TITLE
Use Shell without admin rights on Windows ?

### DISCUSSION THREAD

**Discussion by nthnpxtadcgrp** [moudey/Shell#268] [Q&A]
Hi,
I don't have admin rights on my computer but would highly benefit being able to use Shell on my work computer.
Would there be a known workaround for this use case ?
Best regards,

**Comment by moudey** [comment]
You only need administrator privileges during registration. I will look into this issue.

**Comment by moudey** [comment]
Hi @nthnpxtadcgrp Try this solution

```
Windows Registry Editor Version 5.00

[HKEY_CURRENT_USER\Software\Classes\CLSID\{BAE3934B-8A6A-4BFB-81BD-3FC599A1BAF3}]
@="Nilesoft.Shell"

[HKEY_CURRENT_USER\Software\Classes\CLSID\{BAE3934B-8A6A-4BFB-81BD-3FC599A1BAF3}\InprocServer32]
@="path\\to\\shell.dll"
"ThreadingModel"="Apartment"

[HKEY_CURRENT_USER\Software\Classes\Drive\shellex\FolderExtensions\{BAE3934B-8A6A-4BFB-81BD-3FC599A1BAF3}]
@="Nilesoft.Shell"
"DriveMask"=dword:000000ff
```

### Make sure to write down the path to the "shell.dll" file.

Save this code in an empty file and name it "shell.reg". Double click to merge the file into the Windows Registry and restart the explorer.
